SQL/Archive/2018-05-28_publication_schema.sql

```diff
@@ -0,0 +1,169 @@
+SET FOREIGN_KEY_CHECKS=0;
+-- Publication Status
+DROP TABLE IF EXISTS publication_status;
+CREATE TABLE `publication_status` (
+  `PublicationStatusID` int(2) unsigned NOT NULL AUTO_INCREMENT,
+  `Label` varchar(255) NOT NULL,
+  PRIMARY KEY(`PublicationStatusID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+INSERT INTO publication_status (`Label`) VALUES ('Pending');
+INSERT INTO publication_status (`Label`) VALUES ('Approved');
+INSERT INTO publication_status (`Label`) VALUES ('Rejected');
+
+-- Main table
+DROP TABLE IF EXISTS publication;
+CREATE TABLE `publication` (
+    `PublicationID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `PublicationStatusID` int(2) unsigned NOT NULL default 1,
+    `UserID` int(10) unsigned NOT NULL,
+    `RatedBy` int(10) unsigned,
+    `DateProposed` date NOT NULL,
+    `DateRated` date default NULL,
+    `Title` varchar(255) NOT NULL,
+    `LeadInvestigator` varchar(255) NOT NULL,
+    `LeadInvestigatorEmail` varchar(255) NOT NULL,
```

Comment:
Stand with my previous request (May 29). These 2 attributes (LeadInvestigator and LeadInvestigatorEmail) are the same attributes as in the publication_collaborator table. That last table could be renamed to `person` (or similar) and Investigator attributes be a foreign key to that table.

Surely other features (current or future) in Loris could benefit from that `person` table

Reply:
I agree with you, but i think such a feature is way outside the scope of this PR. Projects (CCNA / IBIS) really need this on the sooner side and i would not want to hold this up with such a big addition. 
However i do think that containing the Lead Investigator fields within the collaborator table with the main publication table referencing that row would be a good idea

Reply:
Ok, that could be done in a future cleanup PR. Will not block for that.